apps/omg_watcher_info/lib/omg_watcher_info/api/stats.ex

```diff
@@ -0,0 +1,67 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherInfo.API.Stats do
+  @moduledoc """
+  Module provides operations related to network statistics.
+  """
+
+  alias OMG.WatcherInfo.DB.{Block, Transaction}
```

Comment:
I haven't seen us using this pattern anywhere else. best to stick with consistency:
```
alias OMG.WatcherInfo.DB.Block
alias OMG.WatcherInfo.DB.Transaction
```

Reply:
Thanks.Done 98fbd15.